sympy/solvers/ode/riccati.py

```diff
@@ -0,0 +1,791 @@
+r"""
+This module contains :py:meth:`~sympy.solvers.ode.riccati.solve_riccati`,
+a function which gives all rational particular solutions to first order
```

Comment:
Is this module included in the rst docs anywhere?

Reply:
No. Could you please tell me where I need to add it? I'm not familiar with rst docs that much.

Reply:
Take a look in `docs/src/modules/solvers/ode.rst`. That's the source code for this page:
https://docs.sympy.org/latest/modules/solvers/ode.html

Reply:
I added some functions to the rst docs. Let me know if I should remove/add any functions.